### PR TITLE
Fix disabled and narrow tabs styling

### PR DESCRIPTION
**Pull request by bafonins**
#### Summary

This PR fixes `disabled` and `narrow` styling of the `<Tabs />` component

#### Changes

- Do not render `<NavLink />` if tab is disabled, otherwise it is still possible to navigate
- Fix `narrow` styles. Styles were not applied to the tab components, as well as the values seemed incorrect.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [x] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/components/tabs/story.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Example extends Component {
 
   render() {
     const { activeTab } = this.state
+    console.log(this.props)
```
Comment (kschiffer) [inline pkg/webui/components/tabs/story.js]:
```suggestion
```

File: pkg/webui/components/tabs/tab/index.js [hunk 2]
```diff
@@ -81,13 +81,18 @@ class Tab extends React.PureComponent {
       [style.tabItemDisabled]: disabled,
     })
 
-    const Component = link ? NavLink : 'span'
+    // There is no support for disabled on anchors in html and hence in `react-router`. So, do
+    // not render the link component if the tab is disabled, but render regular tab item instead.
+    const canRenderLink = link && !disabled
+
```
Comment (kschiffer) [inline pkg/webui/components/tabs/tab/index.js]: Did you try putting the `disabled` prop on the link component? There is a `pointer-events: none` in its css that should prevent navigation.

File: pkg/webui/components/tabs/tab/index.js [hunk 3]
```diff
@@ -81,13 +81,18 @@ class Tab extends React.PureComponent {
...
+    const Component = canRenderLink ? NavLink : 'span'
     const props = {
       role: 'button',
```
Comment (kschiffer) [inline pkg/webui/components/tabs/tab/index.js]:
```suggestion
      disabled,
      role: 'button',
```

See my other comment.

Comment (bafonins) [reply pkg/webui/components/tabs/tab/index.js]: `NavLink` doesnt have the `disabled` prop

File: pkg/webui/components/tabs/tab/index.js (hunk 2 above)
Comment (bafonins) [reply pkg/webui/components/tabs/tab/index.js]: `NavLink` doesnt have the `disabled` prop

Comment (bafonins) [reply pkg/webui/components/tabs/tab/index.js]: Also, `pointer-events` property only disables mouse events which means that it is still possible to navigate through disabled links using the keyboard.

**Pull request merged**